ios/brave-ios/Sources/Brave/Frontend/Reader/Reader.css

```diff
@@ -1,6 +1,7 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
+/* Copyright (c) 2015 All rights reserved.
```

Comment:
i see those outdated licenses in few places, was it automated?
I will skip adding comments about them from now on